.eslintrc.js

```diff
@@ -1,4 +1,8 @@
-{
+let extraPlugins = [];
+if (process.env.STRICT_LINT !== "1")
```

Comment:
needs curly braces.

Reply:
Fixed, and I also moved this to #46, since it really belongs in that PR. I also set eslint to lint this file so it can't happen again.